homeworks/Vladimir.Karbivnychiy_VladimirSkarb/11-dependency-injection/car-list/src/app/app.module.ts

```diff
@@ -0,0 +1,26 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import {appServices} from './services';
+import {appDirectives} from "./directives";
+import {appPipes} from "./pipes";
```

Comment:
Some imports have `'`, some `"`, and some with or without additional space `{ myImport } vs {myImport}`.

Please, make it looks like the same 😄